optuna/importance/_fanova/_fanova.py

```diff
@@ -55,7 +51,7 @@ def __init__(
             random_state=seed,
         )
         self._trees: Optional[List[_FanovaTree]] = None
-        self._variances: Optional[Dict[Tuple[int, ...], numpy.ndarray]] = None
+        self._variances: Dict[int, numpy.ndarray] = {}
         self._column_to_encoded_columns: Optional[List[numpy.ndarray]] = None
```

Comment:
`None` -> `{}` looks like a simplification. At the same time, having only one of the three attributes non-optional seems less clear than the previous logic, as if `self._variances` has a different behavior/semantic. What do you think?

Reply:
Thanks! For me either sounds fine. But I don't need to make it a non-optional attribute at least. I made `self._variances` optional at 72b564a.